pysipp/agent.py

```diff
@@ -39,7 +39,7 @@ class UserAgent(command.SippCmd):
     configuration options for a SIP UA.
     """
     # we skip `error` since we can get it from stderr
-    _log_types = 'screen calldebug message log'.split()
+    _log_types = 'screen calldebug message log error'.split()
```

Comment:
Yes the error file is not always created.
A simple way to produce it, is to add a line in the scenario like <nop next="foo"/> and not define label foo.

I tried to use unit tests but they fail. I don't know why.
ERROR collecting tests/test_stack.py
[...]
/usr/lib/python2.7/site-packages/_pytest/python.py:824: in parametrize
    if ids and len(ids) != len(argvalues):
E   TypeError: object of type 'builtin_function_or_method' has no len()